psi-ms.obo

```diff
@@ -24092,6 +24105,33 @@ relationship: has_domain MS:1002305 ! value between 0 and 1 inclusive
 relationship: has_order MS:1002109 ! lower score better
 synonym: "PG: Contaminants" NARROW []
 
+[Term]
+id: MS:4000180
+name: table of missed cleavage counts
+def: "The number of identified PSM with corresponding number of missed cleavages. The number of missed cleavages per PSM is given in the 'number of missed cleavages' column, the respective count of such PSM in the 'Number of Occurrences' column. The highest 'missed cleavages' row is to be interpreted as that number of missed cleavages or higher." [PSI:MS]
+is_a: MS:4000005 ! table
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
```

Comment:
Somewhat related to the question above. If we're talking about peptides rather than PSMs, is this truly an MS2-level metric? The peptide sequences themselves are not necessarily exclusively derived from MS2 data, e.g. identifications from MS1 during match-between-runs.

Reply:
To the effect of the discussion above, idk about the MS1, but I think it's certainly divorced from the MS2 concept. Just remove the MS2 metric category?

Reply:
Yes, I agree.